src/main/notebooks/tpch_run.scala

```diff
@@ -0,0 +1,79 @@
+// Databricks notebook source
+// TPCH runner (from spark-sql-perf) to be used on existing tables
+// edit the main configuration below
+
+val scaleFactors = Seq(1, 10, 100, 1000) //set scale factors to run
+val format = "parquet" //format has have already been generated
+
+def perfDatasetsLocation(scaleFactor: Int, format: String) = 
+  s"s3a://my-bucket/tpch/sf${scaleFactor}_${format}"
+
+val resultLocation = "s3a://my-bucket/results"
+val iterations = 2
+def databaseName(scaleFactor: Int, format: String) = s"tpch_sf${scaleFactor}_${format}"
+val randomizeQueries = false //to use on concurrency tests
+
+// Experiment metadata for results, edit if outside Databricks
+val configuration = "default" //use default when using the out-of-box config
+val runtype = "Azure Databricks TPCH" + spark.conf.get("spark.databricks.clusterUsageTags.sparkVersion")
```

Comment:
leave out `Azure Databricks` for this example notebook?

Reply:
Removed